app/assets/stylesheets/custom.scss.erb

```diff
@@ -1105,7 +1105,7 @@ a.list-group-item.confirm {
 }
 .panel.active-list {
     border: 1px solid #000 !important;
-    color: #000 !Important;
+    color: #000 !important;
```

Comment:
ideally we would like to avoid the !important, but I understand this is legacy code and maybe outside the work scope, but lets keep in mind if we can improve the code will be better

Reply:
Yeah my CSS lint constantly warned on this so I updated it.